libvast/src/system/index.cpp

```diff
@@ -969,6 +1009,13 @@ index(index_actor::stateful_pointer<index_state> self,
     },
     caf::policy::arg<caf::broadcast_downstream_manager<
       table_slice, vast::type, i_partition_selector>>{});
+  // Read persistent state.
```

Comment:
Should we add a note that this needs to happen after initializing the stream stage because it may start importing new data?

Reply:
I guess the handshake would be blocked until the actor is actually spawned, so probably you're right.